gis/grass/Portfile

```diff
@@ -54,7 +54,8 @@ depends_lib-append  port:bzip2 \
                     port:zlib
 
 patchfiles          01_patch-dyld-link.diff \
-                    02_grass-python.diff
+                    02_grass-python.diff \
+                    03_patch-agl.diff
```

Comment:
is the framewok actually used? If so, you probably want to do this conditionally only on systems where it does not exist anymore?

Reply:
It wasn't actually used, because of missing macro set in config.h. But I fixed that and patched configure to detect presence of AGL framework (but not to fail if not).